palm/pilrc/Portfile

```diff
@@ -16,21 +15,16 @@ long_description	This is a tool that turns a textual description of \
 					prc-tools and CodeWarrior for Palm OS.
 homepage		http://pilrc.sourceforge.net/
 master_sites	sourceforge
-checksums		md5 bbfea963c4e34252f634c6a39a2578b8
+checksums		md5     9a1e114c5fe1f6fa0ffbb742c4d8510e \
```

Comment:
`md5` is no longer necessary - you can remove this line.

The optimal checksums are `rmd160`, `sha256`, and `size`.